src/main/kotlin/com/msg/gauth/domain/client/presentation/ClientController.kt

```diff
@@ -61,7 +65,10 @@ class ClientController(
     }
 
     @PatchMapping("/{id}/patch")
-    fun updateAnyClient(@PathVariable id: Long, @RequestBody clientUpdateDto: ClientUpdateReqDto): ResponseEntity<Void>{
+    fun updateAnyClient(
+        @PathVariable id: Long,
+        @RequestBody clientUpdateDto: ClientUpdateReqDto
+    ): ResponseEntity<Void>{
```

Comment:
```suggestion
    ): ResponseEntity<Void> {
```
스타일 따라서 공백 넣어주면 좋을거같아요